mobile/lib/shared/models/asset.dart

```diff
@@ -222,6 +228,7 @@ class Asset {
   bool canUpdate(Asset a) {
     assert(isInDb);
     assert(checksum == a.checksum);
+    assert(thumbhash == a.thumbhash);
```

Comment:
This is not needed. It might even be wrong. The thumb hash value might even be updated

Reply:
Instead, add the thumb hash comparison to the check in the lines below, so that a changed thumb hash results in this function returning true